web/api/mocks/service/StorageService.js

```diff
@@ -76,9 +76,19 @@ exports.getCollection = function(cName) {
 exports.save = function(cName, id, obj) {
   return new Promise(function(resolve, reject) {
     if (collections[cName]) {
-      let index = _.findIndex(collections[cName], ['id', id]);
-      let item = _.find(collections[cName], ['id', id]);
-      let data = Object.assign({}, item || {}, obj);
+      let index = null;
```

Comment:
I see why this change went in, but as part of our clean-up next sprint, we should change this to be "id", not "jobId".